server/services/assessmentService.ts

```diff
@@ -85,7 +85,7 @@ export default class AssessmentService {
     const responses = getResponses(assessment)
 
     if (!applicationAccepted(assessment)) {
-      return client.rejection(assessment.id, responses, assessment.rejectionRationale)
+      return client.rejection(assessment.id, responses, (responses?.['make-a-decision']?.[0]?.Decision as string) || '')
```

Comment:
Could we have a function here called `extractRejectionRationale` or similar? Also, the use of this `0` index makes me a bit nervous.

Reply:
Yeah sure. 
It does feel a bit risky but I'm not too sure what else we can do to achieve the same result - there is optional chaining to ensure we fallback to an empty string but we could wrap it in a try/catch if you'd prefer

Reply:
Yeah, I guess I don't quite understand why it's coming in as an array is all.

Reply:
OIC - Couldn't we use `getResponseForPage` in that case?